CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [Prerelease] - Unreleased
 
+
+## [v0.11.0] - 2025-11-04
+
+### Fixed
+* Fixes a race condition in killing Sandboxes by @simongdavies in https://github.com/hyperlight-dev/hyperlight/pull/959
+
+### Changed
+* Unify register representation across hypervisors by @ludfjig in https://github.com/hyperlight-dev/hyperlight/pull/907
```

Comment:
This is an internal refactor so not sure it's necessary to mention, but it's fine